README.md

```diff
@@ -70,10 +141,11 @@ android {
         }
     }
 }
-tasks.withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompile> { 
+tasks.withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompile> {
```

Comment:
this part is missing from the groovy example.
It broke some KSP KotlinCompile task if I left it out (the last time I tried it). Maybe it's not needed anymore, but then it should be removed here.

Reply:
Could it be for complex examples with Schema description? I have tested on the example from the readme with dataframe creation without Schema description.

Reply:
Looks like I need a double check here for all possible variants (4 cases), but I need more complex example.
What example did you use to get the broken KSP task?

Reply:
I just used the project at my branch linked below. It used the DataFrame plugin and so KSP to generate @DataSchema accessors. It might be that it broke because the Gradle JVM version was not 8 on my machine and then I got a weird error related to KSP. Adding these lines fixed that. If you cannot reproduce it, it might be fine.

Reply:
Ok, will try with your project, need to double check

Reply:
It's very interesting that this task is really required on Kotlin DSL, but could be ignored (with different JDKs) on the Gradle.  I tested a lot of different combinations. Let's wait something from users here.